src/js/dark-theme.js

```diff
@@ -0,0 +1,41 @@
+if (localStorage.getItem('dark-mode') === 'true') {
+    document.documentElement.classList.add('dark');
+}
+
+document.addEventListener('DOMContentLoaded', function () {
+    const darkModeTogleEl = document.querySelectorAll('.dark-mode');
+    const whiteModeTogleEl = document.querySelectorAll('.white-mode');
+    const darkModeToggle = document.querySelectorAll('.dark-mode-toggler');
+
+    const toggle = (element, value) => {
+        element.forEach((el) => {
+            el.style.display = value
+        });
+    }
+
+
+    if (localStorage.getItem('dark-mode') === 'false') {
+        toggle(whiteModeTogleEl, 'none');
+        toggle(darkModeTogleEl, 'flex')
+    } else {
+        toggle(whiteModeTogleEl, 'flex');
+        toggle(darkModeTogleEl, 'none')
+    }
+
+    darkModeToggle.forEach((el) => {
+        el.addEventListener('click', () => {
+            if (document.documentElement.classList.contains('dark')) {
+                document.documentElement.classList.remove('dark');
+                localStorage.setItem('dark-mode', 'false');
```

Comment:
Just a small tip, but it should be easier to handle values using json stringify and json parse 
```suggestion
                localStorage.setItem('dark-mode', JSON.stringify(false));
```